File: app/src/main/java/com/kelsos/mbrc/commands/ProtocolPongHandle.java
package com.kelsos.mbrc.commands;

import com.kelsos.mbrc.interfaces.ICommand;
import com.kelsos.mbrc.interfaces.IEvent;
import timber.log.Timber;

public class ProtocolPongHandle implements ICommand {
  @Override public void execute(IEvent e) {
    Timber.d(e.getData().toString());
  }
}

File: app/src/main/java/com/kelsos/mbrc/commands/ProtocolPongHandle.kt
package com.kelsos.mbrc.commands

import com.kelsos.mbrc.interfaces.ICommand
import com.kelsos.mbrc.interfaces.IEvent
import timber.log.Timber
import javax.inject.Inject

class ProtocolPongHandle
@Inject constructor() : ICommand {
  override fun execute(e: IEvent) {
    Timber.d(e.data.toString())
  }
}
